internal/config/endpoints.go

```diff
@@ -14,192 +14,97 @@
-		Region:     "europe-west3",
-		RegionName: "Frankfurt",
-	},
-	"europe-west4": {
-		URL:        "https://europe-west4-5tkroniexa-ez.a.run.app",
-		Region:     "europe-west4",
-		RegionName: "Netherlands",
-	},
-	"europe-west6": {
-		URL:        "https://europe-west6-5tkroniexa-oa.a.run.app",
-		Region:     "europe-west6",
-		RegionName: "Zurich",
-	},
-	"europe-west8": {
-		URL:        "https://europe-west8-5tkroniexa-oc.a.run.app",
-		Region:     "europe-west8",
-		RegionName: "Milan",
-	},
-	"europe-west9": {
-		URL:        "https://europe-west9-5tkroniexa-od.a.run.app",
-		Region:     "europe-west9",
-		RegionName: "Paris",
-	},
-	"europe-southwest1": {
-		URL:        "https://europe-southwest1-5tkroniexa-no.a.run.app",
-		Region:     "europe-southwest1",
-		RegionName: "Madrid",
-	},
-	"northamerica-northeast1": {
-		URL:        "https://northamerica-northeast1-5tkroniexa-nn.a.run.app",
-		Region:     "northamerica-northeast1",
-		RegionName: "Montréal",
-	},
-	"northamerica-northeast2": {
-		URL:        "https://northamerica-northeast2-5tkroniexa-pd.a.run.app",
-		Region:     "northamerica-northeast2",
-		RegionName: "Toronto",
-	},
-	"southamerica-east1": {
-		URL:        "https://southamerica-east1-5tkroniexa-rj.a.run.app",
-		Region:     "southamerica-east1",
-		RegionName: "São Paulo",
-	},
-	"southamerica-west1": {
-		URL:        "https://southamerica-west1-5tkroniexa-tl.a.run.app",
-		Region:     "southamerica-west1",
-		RegionName: "Santiago",
-	},
-	"us-central1": {
-		URL:        "https://us-central1-5tkroniexa-uc.a.run.app",
-		Region:     "us-central1",
-		RegionName: "Iowa",
-	},
-	"us-east1": {
-		URL:        "https://us-east1-5tkroniexa-ue.a.run.app",
-		Region:     "us-east1",
-		RegionName: "South Carolina",
-	},
-	"us-east4": {
-		URL:        "https://us-east4-5tkroniexa-uk.a.run.app",
-		Region:     "us-east4",
-		RegionName: "North Virginia",
-	},
-	"us-east5": {
-		URL:        "https://us-east5-5tkroniexa-ul.a.run.app",
-		Region:     "us-east5",
-		RegionName: "Columbus",
-	},
-	"us-south1": {
-		URL:        "https://us-south1-5tkroniexa-vp.a.run.app/",
-		Region:     "us-south1",
-		RegionName: "Dallas",
-	},
-	"us-west1": {
-		URL:        "https://us-west1-5tkroniexa-uw.a.run.app",
-		Region:     "us-west1",
-		RegionName: "Oregon",
-	},
-	"us-west2": {
-		URL:        "https://us-west2-5tkroniexa-wl.a.run.app",
-		Region:     "us-west2",
-		RegionName: "Los Angeles",
-	},
-	"us-west3": {
-		URL:        "https://us-west3-5tkroniexa-wm.a.run.app",
-		Region:     "us-west3",
-		RegionName: "Salt Lake City",
-	},
-	"us-west4": {
-		URL:        "https://us-west4-5tkroniexa-wn.a.run.app",
-		Region:     "us-west4",
-		RegionName: "Las Vegas",
-	},
+// GenerateConfigFromEndpoints is used by the cli to generate an Endpoint map
+// using json served by the gcping endpoints.
+func GenerateConfigFromEndpoints(ctx context.Context, endpointsURL string) (map[string]Endpoint, error) {
```

Comment:
Should this just be `EndpointsFromServer` or something that makes it clear the substance is an HTTP call?